app/packages/operators/src/state.ts

```diff
@@ -83,12 +85,22 @@ export const usePromptOperatorInput = () => {
       params,
       options,
       initialParams: params,
+      id: generateOperatorSessionId(),
     });
   };
 
   return prompt;
 };
 
+/**
+ * The operator session ID is used to identify the operator session in the
+ * backend. It is generated when the operator client is initialized and is
+ * used to track the operator session across different calls to the backend.
+ *
+ * NOTE: the operator session ID is regenerated when the browser is refreshed.
+ */
+export const operatorSessionId = generateOperatorSessionId();
```

Comment:
I'm thinking we actually need to generate and validate this from the server side for security reasons.

Reply:
Going to remove for now, since its a theoretical use case.